lesson02/script.js

```diff
@@ -0,0 +1,12 @@
+const li = document.createElement('li');
+
+const a = document.createElement('a');
+a.href = '1.html';
+a.textContent = 'これです';
+
+const img = document.createElement('img');
+img.src = 'bookmark.png';
+img.alt = 'ブックマーク';
+
+const ul = document.getElementById('list');
+ul.appendChild(li).appendChild(a).prepend(img);
```

Comment:
Because using appendChild with img results in the following DOM.
The text of a and the position of img are reversed.
<li>
  <a href="1.html">これです<img src="bookmark.png" alt="ブックマーク" /></a>
</li>

Reply:
I see!
In this case, it might be more common to use insertAdjacentElement. (I did some research, and I think it's because prepand is not supported by all browsers.) But I don't think movement will be a problem, so I'll leave that up to you.

Reply:
Thank you so much.
I think the general method is good, so I'll give it a try!